src/components/DropDown/DropDown.tsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState } from 'react';
```

Comment:
```suggestion
import { useState } from 'react';
```

안 쓰길래 슬쩍? 지워봤습니다 깃헙에서 알려주니까 편하네요